frontend/src/components/Pages/Register.jsx

```
@@ -1,10 +1,45 @@
+import { useEffect, useState } from "react";
 import pic from "../../assets/img/abt1.jpg";
 import pic2 from "../../assets/img/abt1.png";
 import pic3 from "../../assets/img/abt2.png";
 import pic4 from "../../assets/img/abt3.png";
 import pic5 from "../../assets/img/abt4.png";
 
 export default function Register() {
+  const [Guests, setGuests] = useState();
+  const [Date, setDate] = useState("");
```

Biome

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)